Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 # BEGIN -- workaround lack of go-toolset for golang 1.21
 
-ARG GOLANG_IMAGE=golang:1.21
+ARG GOLANG_IMAGE=docker.io/library/golang:1.21
```

Comment:
Out-of-curiosity, why is that needed?

Reply:
Without this change Podman tries to resolve registry for golang, which failed even though I copied configuration for registries in https://github.com/project-codeflare/codeflare-common/blob/fcf105adb00e1221585404498504ed6f5b53075a/github-actions/nvidia-gpu-setup/action.yml#L13

So I didn't want to dig too deeply into it and rather adjusted image to reference registry too.

Reply:
Ah I see, thanks.